account/src/main/java/org/killbill/billing/account/api/DefaultImmutableAccountData.java

```diff
@@ -59,4 +64,15 @@ public Currency getCurrency() {
     public DateTimeZone getTimeZone() {
         return dateTimeZone;
     }
+
+    @Override
+    public UUID getParentAccountId() {
+        return parentAccountId;
```

Comment:
I see ...
I added those two columns there because I have to check if this is invoice from a child account in ÌnvoiceListener` and I didn't want to got to the database every time an invoice was generated.